tests/ui/mock/alerts_with_videos/alert_summary_with_browsertime_videos.json

```diff
@@ -0,0 +1,271 @@
+{
+  "id": 27115,
+  "push_id": 847217,
+  "prev_push_id": 846998,
+  "created": "2021-01-19T02:30:10.224239",
+  "repository": "autoland",
+  "framework": 13,
+  "alerts": [
+    {
+      "id": 106959,
+      "status": 0,
```

Comment:
I see we only have untriaged alerts. Let's change some alert statuses here, so that the enrichment algorithm covers them as well. Ensure the summary has untriaged, invalid & acknowledged alerts.